client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -65,11 +66,10 @@ class ImageEditorCanvas extends Component {
 	constructor( props ) {
 		super( props );
 
-		this.onWindowResize = null;
-
 		this.onLoadComplete = this.onLoadComplete.bind( this );
+		this.onWindowResize = this.onWindowResize.bind( this );
```

Comment:
You don't need to modify this here, but a heads up that we are allowed to use class properties. https://babeljs.io/docs/plugins/transform-class-properties/

So in the future or in another Janitorial PR PR, we can do:
```jsx
onWindowResize = () => {
    //...
}
```

Rather than binding in the constructor.